### ISSUE TITLE
EF Core 2.2.4 doesn't always seem to copy Microsoft.Extensions.DependencyInjection.Abstractions to bin folder in .Net projects

### ISSUE THREAD

**Issue by JRBonnema** [dotnet/efcore#15903]
I use .Net projects that make use of EF Core.  Since the new version 2.2.4 of EF Core in some projects (not all) I encounter a new issue that the object Microsoft.Extensions.DependencyInjection.Abstractions.dll (11-11-2018) and its xml counterpart are not copied to the bin folder when compiling the project. I tried some things but until now the only way to solve this problem I found is manually copying the two objects to the bin folder. I'm relatively sure (lets say 90%) that the problem is new and is in some vague way related to the introduction of EF 2.2.4
```

### Further technical details
EF Core 2.2.4.version: 
Database Provider: (e.g. Microsoft.EntityFrameworkCore.SqlServer)
Operating system:  IDE: (e.g. Visual Studio 2019 16.1)

**Comment by ajcvickers** [comment]
@JRBonnema Can you post a project/solution that demonstrates what you are seeing so that we can investigate?

**Comment by JRBonnema** [comment]
@acjvickers : 
[ApplicationControlDMLibrary.zip](https://github.com/aspnet/EntityFrameworkCore/files/3249808/ApplicationControlDMLibrary.zip)

**Comment by JRBonnema** [comment]
I see that I mistakenly closed the issue. See if I can reopen the issue by posting this comment

**Comment by ajcvickers** [comment]
@JRBonnema Restores and builds fine for me in VS2017 and VS2019, and the assemblies are coped to bin. What might be different on your machine?

**Comment by JRBonnema** [comment]
Just like you did, I downloaded the zipped project that I uploaded to Github yesterday and created a new project with it. There is indeed something different on my machine since now I get two MSB3245 warnings. In addition to Microsoft.Extensions.DependencyInjection.Abstractions it can also not find Microsoft.Extensions.Configuration.Abstractions.  I did a more extensive internet search to see if other people have encountered similar problems in the last month. This is in not the case. I will close the issue. Thanks for your time.
